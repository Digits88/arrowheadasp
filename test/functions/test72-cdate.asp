Test CDate function

CDate("1/12/10") = 
    <%= CDate("1/12/10") %><br>
CDate("1/12/90") = 
    <%= CDate("1/12/90") %><br>
CDate("1:30") = 
    <%= CDate("1:30") %><br>
CDate("1:30:15") = 
    <%= CDate("1:30:15") %><br>
CDate("15:30:15") = 
    <%= CDate("15:30:15") %><br>
CDate("January 12 1:30:15") = 
    <%= CDate("January 12 1:30:15") %><br>
CDate("January 12 15:30:15") = 
    <%= CDate("January 12 15:30:15") %><br>
CDate("January 12, 1920 1:30:15") = 
    <%= CDate("January 12, 1920 1:30:15") %><br>
CDate("Jun 25, 1920 2:30 pm") = 
    <%= CDate("Jun 25, 1920 2:30 pm") %><br>
