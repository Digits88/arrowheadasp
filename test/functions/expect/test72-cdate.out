Test CDate function

CDate("1/12/10") = 
    1/12/2010<br>
CDate("1/12/90") = 
    1/12/1990<br>
CDate("1:30") = 
    1:30:00 AM<br>
CDate("1:30:15") = 
    1:30:15 AM<br>
CDate("15:30:15") = 
    3:30:15 PM<br>
CDate("January 12 1:30:15") = 
    1/12/^[0-9]{4}$ 1:30:15 AM<br>
CDate("January 12 15:30:15") = 
    1/12/^[0-9]{4}$ 3:30:15 PM<br>
CDate("January 12, 1920 1:30:15") = 
    1/12/1920 1:30:15 AM<br>
CDate("Jun 25, 1920 2:30 pm") = 
    6/25/1920 2:30:00 PM<br>
